### PR TITLE
ColoredConsoleTarget.cs: Fix typo

### PR DESCRIPTION
**Pull request by perlun**
Also improved the general grammar of the comment a bit.

### CODE REVIEW

File: src/NLog/Targets/ColoredConsoleTarget.cs [hunk 1]
```diff
@@ -299,14 +299,14 @@ private void WriteToOutput(LogEventInfo logEvent, string message)
                 }
                 catch (IndexOutOfRangeException ex)
                 {
-                    //this is a bug and therefor stopping logging. For docs, see PauseLogging property
+                    // This is a bug and will therefore stopp the logging. For docs, see the PauseLogging property.
```
Comment (304NotModified) [inline src/NLog/Targets/ColoredConsoleTarget.cs]: isn't it "stop"?

Comment (perlun) [reply src/NLog/Targets/ColoredConsoleTarget.cs]: Ah, my bad... of course! Thanks, will fix.

**Comment by perlun** [comment]
@304NotModified Thanks for the feedback, please re-check.

My editor trimmed a lot of trailing whitespace when I saved this file - I hope this is OK. If not, let me know and I can cherry-pick that change out to a separate PR, if you so prefer.

**Comment by 304NotModified** [comment]
thanks, please cherry pick. Force push is OK (but also OK without force push, then I will squash it).

**Comment by perlun** [comment]
> thanks, please cherry pick. Force push is OK (but also OK without force push, then I will squash it).

Force-push is fine, it's one of the premiums we get for choosing git, isn't it. 😉 There you have it. I'll take the whitespace stuff in a separate PR: #2584

**Comment by 304NotModified** [comment]
Thanks!

Git _o_

**Pull request merged**